sys-devel/gettext/gettext-0.19.7.recipe

```diff
@@ -97,7 +107,6 @@ INSTALL()
 	make install AUTOCONF=: AUTOMAKE=: AUTOHEADER=: ACLOCAL=:
 
 	rm $libDir/charset.alias
-	rm $libDir/lib*.la
```

Comment:
OK, thanks for the template. I'm not running Haiku at the moment (and the PC I run Haiku on is currently stuck with a biiiiiig update of openSUSE) but I'll try to update my commit to include a fix for the **`lib*.la`** files.